netket/nn/symmetric_linear.py

```diff
@@ -661,8 +700,8 @@ def DenseSymm(symmetries, point_group=None, mode="auto", shape=None, **kwargs):
         features: The number of output features. The full output shape
             is :code:`[n_batch,features,n_symm]`.
         use_bias: A bool specifying whether to add a bias to the output (default: True).
-        mask: An optional array of shape [n_sites] consisting of ones and zeros
-            that can be used to give the kernel a particular shape.
+        input_mask: Optional array of shape [n_sites] that used to restrict the convolutional
```

Comment:
proposal: why not calling this `mask` instead of `input mask`? 
For the DenseSymm there is no other mask so no need to discern the two... or am I wrong?